frontend/app/equity/dividend_computations/new/page.tsx

```
@@ -0,0 +1,257 @@
+      await new Promise(resolve => setTimeout(resolve, 1000)); // Simulate API call
+      
+      // Validate amount
+      const parsedAmount = parseFloat(amount);
+      if (isNaN(parsedAmount) || parsedAmount <= 0) {
+        setError('Please enter a valid amount greater than 0');
+        return;
+      }
+      
+      // Generate a unique demo ID based on timestamp to avoid collisions
+      const existingComputations = JSON.parse(localStorage.getItem('demoComputations') || '[]');
+      const existingIds = new Set(existingComputations.map((comp: any) => comp.id));
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)